10/s2i/bin/assemble

```diff
@@ -27,6 +27,11 @@ safeLogging () {
 }
 
 shopt -s dotglob
+if [ -d /tmp/artifacts ] && [ "$(ls /tmp/artifacts/ 2>/dev/null)" ]; then
+    echo "---> Restoring previous build artifacts ..."
+    mv -T --verbose /tmp/artifacts/node_modules "${HOME}/node_modules"
```

Comment:
Is the `--verbose` flag supposed to be here? It generates an awful amount of output ...

Reply:
I did add it intentionally to get a "log" that something actually happens; and when I test it locally (built with `VERSION=8 TARGET=centos7`), it produces one line:

```
$ s2i build --incremental https://github.com/sclorg/nodejs-ex s2i-nodejs8-incremental hello-node8
...snip...
---> Restoring previous build artifacts ...
'/tmp/artifacts/node_modules' -> '/opt/app-root/src/node_modules'
---> Installing application source ...
...snip...
```

However, if you are getting lots of output (I assume it is reporting *every* file it moves), it can be safely removed.